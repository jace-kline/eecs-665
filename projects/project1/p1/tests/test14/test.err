FATAL [38,1]: Illegal character &
