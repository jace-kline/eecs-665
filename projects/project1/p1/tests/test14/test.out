INT [1,1]
INTPTR [2,1]
BOOL [3,1]
BOOLPTR [4,1]
CHAR [5,1]
CHARPTR [6,1]
VOID [7,1]
IF [8,1]
ELSE [9,1]
WHILE [10,1]
RETURN [11,1]
TRUE [12,1]
FALSE [13,1]
FROMCONSOLE [14,1]
TOCONSOLE [15,1]
NULLPTR [16,1]
AT [17,1]
CARAT [18,1]
LBRACE [19,1]
RBRACE [20,1]
LCURLY [21,1]
RCURLY [22,1]
LPAREN [23,1]
RPAREN [24,1]
SEMICOLON [25,1]
COMMA [26,1]
DASHDASH [27,1]
DASH [28,1]
CROSS [29,1]
CROSSCROSS [30,1]
STAR [31,1]
SLASH [32,1]
NOT [33,1]
EQUALS [34,1]
NOTEQUALS [35,1]
LESS [36,1]
GREATER [37,1]
LESSEQ [39,1]
GREATEREQ [40,1]
ASSIGN [41,1]
AND [42,1]
OR [43,1]
EOF [43,3]
